packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -54,16 +67,41 @@ export async function handleConfirmation(args: {
 
     log.error(`Booking ${user.username} failed`, JSON.stringify({ error, results }));
   } else {
-    const metadata: AdditionalInformation = {};
-
     if (results.length) {
       // TODO: Handle created event metadata more elegantly
       metadata.hangoutLink = results[0].createdEvent?.hangoutLink;
       metadata.conferenceData = results[0].createdEvent?.conferenceData;
       metadata.entryPoints = results[0].createdEvent?.entryPoints;
     }
     try {
-      await sendScheduledEmails({ ...evt, additionalInformation: metadata });
+      const eventType = booking.eventType;
+      const eventTypeMetadata = EventTypeMetaDataSchema.parse(eventType?.metadata || {});
+      let isHostConfirmationEmailsDisabled = false;
+      let isAttendeeConfirmationEmailDisabled = false;
+
+      const workflows = eventType?.workflows?.map((workflow) => workflow.workflow);
+
+      if (workflows) {
+        isHostConfirmationEmailsDisabled =
+          eventTypeMetadata?.disableStandardEmails?.confirmation?.host || false;
+        isAttendeeConfirmationEmailDisabled =
+          eventTypeMetadata?.disableStandardEmails?.confirmation?.attendee || false;
+
+        if (isHostConfirmationEmailsDisabled) {
+          isHostConfirmationEmailsDisabled = allowDisablingHostConfirmationEmails(workflows);
+        }
+
+        if (isAttendeeConfirmationEmailDisabled) {
+          isAttendeeConfirmationEmailDisabled = allowDisablingAttendeeConfirmationEmails(workflows);
+        }
+      }
+
+      await sendScheduledEmails(
```

Comment:
Make sure confirmation emails are not sent when disabled because of an existing workflow